scripts/jenkins/build

```diff
@@ -52,6 +55,29 @@ fi
 
 ./autogen.sh |& tee $ARTIFACTS/configure.log
 
+
```

Comment:
if [[ "$TRANSPORT" == "lnet" ]]; then
       TRANSPORT = 'lustre-client'

Reply:
why not use a new name such as:
```
if [[ "$TRANSPORT" == "lnet" ]]
then
	PACKAGE='lustre-client'
else
	echo "No need chage value"
fi